node/tests/http_through_node_test.rs

```diff
@@ -14,7 +15,7 @@ use std::time::Duration;
 fn http_through_node_integration() {
     let _node = utils::MASQNode::start_standard(
         "http_through_node_integration",
-        None,
+        Some(CommandConfig::new().pair("--blockchain-service-url", "https://booga.com")),
```

Comment:
If you modify `standard_args()` as mentioned above, this shouldn't be necessary.